planner/core/exhaust_physical_plans.go

```diff
@@ -1071,6 +1071,7 @@ func (p *LogicalJoin) constructInnerIndexScanTask(
 			Columns:         ds.Columns,
 			Table:           is.Table,
 			TableAsName:     ds.TableAsName,
+			DBName:          ds.DBName,
```

Comment:
It's missed and may output the wrong table name in the explain result?

Reply:
It's not quite related to this PR. It won't cause the wrong table name, just sometimes the `DBName` field is missing.
We don't output DB names in EXPLAIN, but I'll use them in the "access object" in the next PR. So I just fixed it here for convenience.